homeassistant/components/overkiz/cover_entities/vertical_cover.py

```diff
@@ -0,0 +1,111 @@
+"""Support for Overkiz Vertical Covers."""
+from __future__ import annotations
+
+from typing import Any, Union, cast
+
+from pyoverkiz.enums import OverkizCommand, OverkizState, UIClass, UIWidget
+
+from homeassistant.components.cover import (
+    ATTR_POSITION,
+    DEVICE_CLASS_AWNING,
```

Comment:
The old device class constants are deprecated. We have the enum instead.

https://github.com/home-assistant/core/blob/dc1db463fc7fe44e8f642b000d5c51253671cbe7/homeassistant/components/cover/__init__.py#L50